### DISCUSSION TITLE
Cubit Model I can Borrow

### DISCUSSION THREAD

**Discussion by Andres-fierro-inl** [idaholab/moose#25003] [Q&A Meshing]
Does anyone have a Cubit mesh of a 

Cylinder with an extruding pipe from the middle and bottom?

I do not know how to mesh it.

I appreciate you all

**Comment by GiudGiud** [comment]
@GregVernon @rwcarlsen if you could please point us to the right documentation or example for meshing this. 
thank you!

**Reply by GregVernon** [reply]
Sure, I'm not sure I completely understand the request, but tomorrow I'll work up an example.

**Reply by GiudGiud** [reply]
@Andres-fierro-inl can you please post a picture of the relevant part of the cad

**Reply by Andres-fierro-inl** [reply]
I lost track of this, Sorry!

I got it figured out, though! 

Thank you both
